client/src/components/BackToTop/BackToTop.tsx

```diff
@@ -9,21 +9,34 @@ import { is_mobile } from "src/core/feature_detection";
 
 import "./BackToTop.scss";
 
-export class BackToTop extends React.Component {
-  constructor(props) {
+interface BackToTopProps {
+  focus: Function;
```

Comment:
Could be `() => void` I think. That should make it clear that the `focus` function is just for side effects.